internal/devserver/package.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "@bazel/typescript-devserver",
```

Comment:
we won't ever publish this to npm. how about just `description` and `devDependencies`

also could you create `internal/tsc_wrapped/package.json` containing
```
{
  "description": "runtime dependencies to compile and run tsc_wrapped",
  "devDependencies": {
    "@types/node": "7.0.18",
    "@types/source-map": "^0.5.1",
    "protobufjs": "5.0.0",
    "tsickle": "0.25.x",
    "typescript": "2.5.x",
    "tsutils": "2.12.1"
  }
}
```
and point the other `npm_install` rule there?

Reply:
with exact versions please

Reply:
Sure. vscode complains about missing "name" and "version" in package.json but no biggie. Npm install will probably issue a warning as well. Will check.

Reply:
`@types/jasmine` required as well. I have these versions ```    "@types/jasmine": "2.8.2",
    "@types/node": "7.0.18",
    "@types/source-map": "0.5.2",
    "protobufjs": "5.0.0",
    "tsickle": "0.25.5",
    "typescript": "2.5.3",
    "tsutils": "2.12.1"
```

Reply:
There are newer versions available `@types/node` and `tsutils` and there is a 5.0.1 release of `protobufjs`